modules/visual_script/visual_script_editor.cpp

```diff
@@ -627,7 +627,7 @@ void VisualScriptEditor::_update_graph(int p_only_id) {
 		Control::get_theme_icon("Quaternion", "EditorIcons"),
 		Control::get_theme_icon("AABB", "EditorIcons"),
 		Control::get_theme_icon("Basis", "EditorIcons"),
-		Control::get_theme_icon("Transform", "EditorIcons"),
+		Control::get_theme_icon("Transform3D", "EditorIcons"),
 		Control::get_theme_icon("Color", "EditorIcons"),
```

Comment:
Not sure about this one, @vnen , @aaronfranke are these icons renamed? I noticed there are a bunch more like this in `./editor`

Reply:
`Transform.svg` hasn't been renamed, would be good to fix it too in this PR:
```
git mv editor/icons/Transform.svg editor/icons/Transform3D.svg
```

Reply:
Also found:
```
editor/connections_dialog.cpp
449:    type_list->add_item("Transform", Variant::TRANSFORM3D);

editor/property_selector.cpp
138:                    search_options->get_theme_icon("Transform", "EditorIcons"),
```
And plenty of `"Transform"` in `editor/plugins/visual_shader_editor_plugin.cpp` (some referring to the icon, others might be for different stuff).

Reply:
The SVG file will need to be renamed.